src/bend.ts

```diff
@@ -1,15 +1,12 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
-//
-// ## Description
-//
-// This file implements tablature bends.
+// MIT License
 
 import { RuntimeError } from './util';
 import { Flow } from './flow';
 import { Modifier } from './modifier';
 import { ModifierContextState } from './modifiercontext';
 
-export interface BendPhrase {
+interface BendPhrase {
```

Comment:
This should be exported, it's part of the API (in constructor.)

Reply:
done